### ISSUE TITLE
Add "Take All" Button to all loot containers.

### ISSUE THREAD

**Issue by BananaMan62** [EpochModTeam/Epoch#946]
Can a "Take All' Button be added to ALL crates & large inventory items to make it easier for looting the same as you have for the underwater crates please?

**Comment by vbawol** [comment]
I agree, I know we talked about it before the release of 1.0 and wanted to perfect it on the underwater crates first. So, I think this is something we could do for 1.1. It can use the dyna action menu and some form of the existing code.
